spec/openapi.yaml

```diff
@@ -732,6 +725,25 @@ paths:
           description: No Content
         403:
           $ref: "#/components/responses/Unauthorized"
+    delete:
+      summary: Terminate the current file download process.
+      responses:
+        204:
+          description: No Content
+                       No Error
+
+  /api/download/{target}:
```

Comment:
[question]
We also have `/api/downloads/{target}`. Aren't we doing something wrong?

Reply:
`/api/downloads` this endpoint is octoprint thing. They even do not mention it as separate endpoint. https://docs.octoprint.org/en/master/api/files.html?highlight=downloads. It is referenced in openapi.yaml only for completeness. AnywayIt is used for downloading files from printer to browser. Simply when you click on download button in project list.

`/api/download` as I understand is used for remote URL download (URL -> printer)

Reply:
I'm no expert on format of openapi.yaml. But GET /api/download is used without any other parameter so it seemed more logical to separate GET https://github.com/prusa3d/Prusa-Link-Web/blob/8609d746b92348a685d7bb26ee61db3c91422e8c/spec/openapi.yaml#L688 and POST https://github.com/prusa3d/Prusa-Link-Web/blob/8609d746b92348a685d7bb26ee61db3c91422e8c/spec/openapi.yaml#L735 to two endpoints